### PR TITLE
Add Windows package to the packaging section

### PR DESCRIPTION
**Pull request by aminya**
Fixes https://github.com/emscripten-core/emsdk/issues/272

I added the packages to the bottom of the downloads page, instead of a hidden mention in a very large note.

### CODE REVIEW

**Review by sbc100 — APPROVED** [review]

**Comment by sbc100** [comment]
Thanks!  lgtm.

File: site/source/docs/getting_started/downloads.rst [hunk 1]
```diff
@@ -201,3 +189,38 @@ The entire Emscripten SDK is also available in the form of a `docker image
...
+ - package info: https://formulae.brew.sh/formula/emscripten
+ - maintainer: @chenrui333
+
+**Arch Linux**
+ - package info: https://github.com/archlinux/svntogit-community/tree/packages/emscripten/trunk
+ - maintainer: Sven-Hendrik Haase <svenstaro@gmail.com>
+
+**Gentoo Linux** (custom overlay)
+ - package info: `dev-util/emscripten` in [darthgandalf-overlay](https://github.com/DarthGandalf/gentoo-overlay)
+ - maintainer: @DarthGandalf
```
Comment (sbc100) [inline site/source/docs/getting_started/downloads.rst:226]: I wonder if it makes sense to duplicate this  information here.   I originally put this info in packaging.md so we have somewhere to internally coordinate with packagers .. but I can see that this is could be useful to end users too, although perhaps not all of it.

If we do move it here we should probably remove it from the other location and replace that with a link.

Comment (aminya) [reply site/source/docs/getting_started/downloads.rst:226]: I can remove it from the packaging.md

Comment (aminya) [reply site/source/docs/getting_started/downloads.rst:225]: Fixed in https://github.com/emscripten-core/emscripten/pull/13149/commits/6fba55b61bf4c1f7930ca320a2a5c88b6377edd3

**Comment by kripken** [comment]
I think this is ready to land? But please add yourself to AUTHORS first @aminya  Thanks!

**Comment by aminya** [comment]
> I think this is ready to land? But please add yourself to AUTHORS first @aminya Thanks!

Done. You're welcome!

**Pull request merged**